### ISSUE TITLE
custom_channels incorrect in 'conda config --show'

### ISSUE THREAD

**Issue by kalefranz** [conda/conda#6509]
Especially for anaconda channel, weird replacement happening.

**Comment by kalefranz** [comment]
solution merged to 4.3.x

**Comment by jgmorrison** [comment]
This does not appear to be fixed. I have a .condarc file with the following:
```
channel_alias: http://anacondarepo.fakesite.net:8080/conda/
ssl_verify: true
default_channels:
  - http://anacondarepo.fakesite.net:8080/conda/anaconda
  - http://anacondarepo.fakesite.net:8080/conda/r-channel
  - http://anacondarepo.fakesite.net:8080/conda/msys2
  - http://anacondarepo.fakesite.net:8080/conda/conda-forge
  - http://anacondarepo.fakesite.net:8080/conda/Research

channels:
  - defaults
```

This makes the custom_channels section show 

```
anaconda: http://repo.fakesite.net:8080/conda/
r-channel: http://anacondarepo.fakesite.net:8080/conda/
msys2: http://anacondarepo.fakesite.net:8080/conda/
conda-forge: http://anacondarepo.fakesite.net:8080/conda/
Research: http://anacondarepo.fakesite.net:8080/conda/
```

So there is something where it extracts "anaconda" from the URL for the key anaconda.

**Comment by kalefranz** [comment]
https://github.com/conda/conda/pull/6510

**Comment by kalefranz** [comment]
@jgmorrison What's `conda info`?

**Comment by jgmorrison** [comment]
```
     active environment : None
       user config file : C:\Users\builder\.condarc
 populated config files : C:\Users\builder\.condarc
          conda version : 4.4.6
    conda-build version : 3.0.27
         python version : 3.6.3.final.0
       base environment : C:\Users\builder\Anaconda3  (writable)
           channel URLs : http://anacondarepo.fakesite.net:8080/conda/anaconda/win-64
                          http://anacondarepo.fakesite.net:8080/conda/anaconda/noarch
                          http://anacondarepo.fakesite.net:8080/conda/r-channel/win-64
                          http://anacondarepo.fakesite.net:8080/conda/r-channel/noarch
                          http://anacondarepo.fakesite.net:8080/conda/msys2/win-64
                          http://anacondarepo.fakesite.net:8080/conda/msys2/noarch
                          http://anacondarepo.fakesite.net:8080/conda/conda-forge/win-64
                          http://anacondarepo.fakesite.net:8080/conda/conda-forge/noarch
                          http://anacondarepo.fakesite.net:8080/conda/Research/win-64
                          http://anacondarepo.fakesite.net:8080/conda/Research/noarch
          package cache : C:\Users\builder\Anaconda3\pkgs
                          C:\Users\builder\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\builder\Anaconda3\envs
                          C:\Users\builder\AppData\Local\conda\conda\envs
                          C:\Users\builder\.conda\envs
               platform : win-64
             user-agent : conda/4.4.6 requests/2.18.4 CPython/3.6.3 Windows/7 Windows/6.1.7601
          administrator : False
             netrc file : None
           offline mode : False
```

**Comment by kalefranz** [comment]
Can you try with `4.3.32`? I could have been wrong that the solution was merged up to 4.4.x.

**Comment by kalefranz** [comment]
Yeah so totally my bad.  We haven't released 4.3.32 yet. Doh.

I've just done the merge up of 4.3.x to 4.4.x.  It'll for sure be in 4.4.7.  It'll also be in 4.3.32, once it's released.

**Comment by JeremyEnns** [comment]
This issue appears to still be happening in 4.3.32 and 4.4.7 for the customer.
